Widget/app/src/test/java/com/microsoft/device/display/samples/widget/ParserRssFeedTest.kt

```diff
@@ -0,0 +1,178 @@
+package com.microsoft.device.display.samples.widget
+
+import android.text.Html
+import android.text.SpannableString
+import com.microsoft.device.display.samples.widget.feed.RssFeed
+import com.microsoft.device.display.samples.widget.feed.RssItem
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.mockkStatic
+import io.mockk.verify
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class ParserRssFeedTest {
+
+    init {
+        mockkObject(RssFeed)
+    }
+
+    @After
+    fun resetMocks() {
+        clearAllMocks()
+    }
+
+    @Test
+    fun shouldInsertItemInRssItems_shouldReturnTrue_whenTagIsLowercaseAndItemNotNull() {
+        val mockTag = RssFeed.ITEM
```

Comment:
Probably you could use a different naming for data, so instead of `mockTag` it could be just `tag` or `dummyTag`.
When you use `mock` to me is that you are mocking a functionality that can be mocked, when here what you do is just to use variables that define a content.

Reply:
You're right, removed the mock namings